src/core/subdirdata.cpp

```diff
@@ -428,7 +423,7 @@ namespace mamba
 
     MRepo MSubdirData::create_repo(MPool& pool)
     {
-        RepoMetadata meta{ m_url,
+        RepoMetadata meta{ m_index_url,
```

Comment:
maybe it would be better to use `repodata_json_url` or `repodata_url`?

Reply:
if we prefer to use a conda semantic repodata = package index, yes.
I can refactor that

Reply:
otherwise it's just more confusing in the end since people might wonder what the index is :)